samples/Azure.Management.Storage/readme.md

```diff
@@ -43,6 +43,9 @@ rename-mapping:
   BlobServiceProperties: BlobService
   FileServiceProperties: FileService
 
+enable-lro-interim-status:
```

Comment:
This also works with the swagger setting right?  If so can we simply do that directive here instead and remove this configuration as eventually we want this to bubble up into the swagger / cadl and not live in customization.

Reply:
I changed the implementation to use the [swagger extension setting](https://github.com/Azure/autorest.csharp/blob/9a3032f7bc84eeb8656e97da0c7ae7a53c9c05cb/src/AutoRest.CSharp/Common/Input/CodeModelPartials.cs#L43).